src/createAdder.js

```diff
@@ -20,7 +20,13 @@
  * @param {number} initialValue
  */
 function createAdder(initialValue = 0) {
-  // write code here
+  const addedValues = [initialValue];
```

Comment:
не усложняй названия переменных, делай их простыми
```suggestion
  const values = [initialValue];
```